### PR TITLE
Adding jwt-cpp version 0.7.0.bcr.1

### PR DESCRIPTION
**Pull request by olipauly**
Visibility is added to an additional target.

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (jwt-cpp) have been updated in this PR. Please review the changes.

File: modules/jwt-cpp/0.7.0.bcr.1/source.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "url": "https://github.com/Thalhammer/jwt-cpp/archive/refs/tags/v0.7.0.tar.gz",
+    "integrity": "sha256-uesnDjuoIh5LK8OHI8mhy0+mwkGkKQi5ozTa/zETdAY=",
+    "strip_prefix": "jwt-cpp-0.7.0",
+    "patch_strip": 1,
+    "patches": {
+        "add-build.patch": "sha256-893niHcbMlUl7JFb82Wr3tmirBTWi5pSUQQR64pnTIo=",
```
Comment (udaya2899) [inline modules/jwt-cpp/0.7.0.bcr.1/source.json:7]: For the patches that don't change from `0.7.0`, you can just use the relative path here, like "add-build.patch": ../0.7.0/patches/add-build.patch" except for the patch that actually changes

Comment (udaya2899) [reply modules/jwt-cpp/0.7.0.bcr.1/source.json:7]: Sorry, it's instead like this: https://github.com/bazelbuild/bazel-central-registry/blob/main/modules/osqp-cpp/0.0.0-20231004-4343373.bcr.1/source.json#L6

Comment (olipauly) [reply modules/jwt-cpp/0.7.0.bcr.1/source.json]: Thanks, I adapted it.

**Review by udaya2899 — APPROVED** [review]
LGTM

**Review by meteorcloudy — APPROVED** [review]

**Pull request merged**